packages/manager/src/features/Profile/APITokens/utils.test.ts

```diff
@@ -304,9 +315,10 @@ describe('APIToken utils', () => {
         ];
         expect(allScopesAreTheSame(scopes)).toBe(2);
       });
-      it('should return null if all scopes are different', () => {
+      it('should return null if all scopes are not same', () => {
```

Comment:
do we need change the name here?, is it even required for an already existing message

Reply:
It is very confusing, it says all scopes are different which is wrong literally. It should be "not same".

Reply:
If its existing message, better not to change.